packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -44,10 +48,43 @@ export const ContextMenu: React.FC<ContextMenuProps> = React.memo(({ x, y, onClo
     }
   });
 
+  const onClick = useCallback(() => {
+    if (onClose) {
+      onClose();
+    }
+  }, [onClose]);
+
   const header = renderHeader && renderHeader();
   return (
     <Portal>
-      <Menu header={header} items={items} onClose={onClose} ref={menuRef} style={positionStyles} />
+      <Menu
+        header={header}
+        ref={menuRef}
+        style={positionStyles}
+        ariaLabel={selectors.components.Menu.MenuComponent('Context')}
+      >
+        {itemsGroup &&
+          itemsGroup.map((group) => (
+            <MenuGroup key={group.label} label={group.label} ariaLabel={group.label}>
+              <List
+                items={group.items || []}
+                renderItem={(item) => {
+                  return (
```

Comment:
This List is totally pointless here, just do group.items.map instead  (see how I removed the same in ButtonSelect here: 
https://github.com/grafana/grafana/pull/31639/commits/42d018ade00ecdff8261167dad39e503df630b20

Reply:
Alright 👍

Reply:
but fixed that in latest commit

Reply:
awesome! Thanks Torkel